airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/commands.py

```diff
@@ -114,7 +108,19 @@ def validate_environment(is_local: bool, use_remote_secrets: bool):
         )
 
 
-@click.group(help="Commands related to connectors and connector acceptance tests.")
+@click.group(
+    cls=LazyGroup,
+    help="Commands related to connectors and connector acceptance tests.",
+    lazy_subcommands={
+        "build": "pipelines.airbyte_ci.connectors.build_image.commands.build",
+        "test": "pipelines.airbyte_ci.connectors.test.commands.test",
+        "list": "pipelines.airbyte_ci.connectors.list.commands.list",
+        "publish": "pipelines.airbyte_ci.connectors.publish.commands.publish",
+        "bump_version": "pipelines.airbyte_ci.connectors.bump_version.commands.bump_version",
+        "migrate_to_base_image": "pipelines.airbyte_ci.connectors.migrate_to_base_image.commands.migrate_to_base_image",
+        "upgrade_base_image": "pipelines.airbyte_ci.connectors.upgrade_base_image.commands.upgrade_base_image",
```

Comment:
This makes me sad because its more things the IDE can't refactor for us 😥

Reply:
I know :( 

This was a solution recommended by click itself so I went with it

Reply:
Understandable!